mmv1/third_party/terraform/services/cloudquotas/resource_cloud_quotas_quota_preference_test.go

```diff
@@ -0,0 +1,195 @@
+package cloudquotas_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccCloudQuotasQuotaPreference_cloudquotasQuotaPreferenceBasicExample_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":          envvar.GetTestOrgFromEnv(t),
+		"billing_account": envvar.GetTestBillingAccountFromEnv(t),
+		"random_suffix":   acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"time": {},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCloudQuotasQuotaPreference_cloudquotasQuotaPreferenceBasicExample_basic(context),
+			},
+			{
+				ResourceName:            "google_cloud_quotas_quota_preference.my_preference",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"parent", "quota_preference_id", "ignore_safety_checks", "contact_email"},
+			},
+			{
+				Config: testAccCloudQuotasQuotaPreference_cloudquotasQuotaPreferenceBasicExample_increaseQuota(context),
+			},
+			{
+				ResourceName:            "google_cloud_quotas_quota_preference.my_preference",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"parent", "quota_preference_id", "ignore_safety_checks", "contact_email", "justification", "quota_config.0.annotations"},
+			},
+			{
+				Config: testAccCloudQuotasQuotaPreference_cloudquotasQuotaPreferenceBasicExample_decreaseQuota(context),
+			},
+			{
+				ResourceName:            "google_cloud_quotas_quota_preference.my_preference",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"parent", "quota_preference_id", "ignore_safety_checks", "contact_email", "justification", "quota_config.0.annotations"},
+			},
+		},
+	})
+}
+
+func testAccCloudQuotasQuotaPreference_cloudquotasQuotaPreferenceBasicExample_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+		resource "google_project" "new_project" {
+			project_id 		= "tf-test%{random_suffix}"
+			name       		= "tf-test%{random_suffix}"
+			org_id			= "%{org_id}"
+			billing_account	= "%{billing_account}"
+		}
+
+		resource "google_project_service" "cloudquotas" {
+			project  	= google_project.new_project.project_id
+			service 	= "cloudquotas.googleapis.com"
+			depends_on	= [google_project.new_project]
+		}
+
+		resource "google_project_service" "compute" {
+			project  	= google_project.new_project.project_id
+			service 	= "compute.googleapis.com"
+			depends_on	= [google_project.new_project]
```

Comment:
I was not aware of this, but when multiple `google_project_service` resources are used, they are actually batched into a single request, and that appears to cause issues with our VCR recording. I think you will need to chain these as an alternative unfortunately (see https://github.com/GoogleCloudPlatform/magic-modules/pull/7282 as an example).